docs/operate/mobility/move-arm/frame-how-to.md

```diff
@@ -0,0 +1,96 @@
+---
+
+Frames can be confusing.
+This guide will help you decide on a coordinate system for your workspace, and then determine how the frames of your arm and other components relate to that workspace.
+
+Once you have configured frames for all your components, Viam will keep track of the positions and orientations as your robot moves, so you can plan motion in terms of a consistent coordinate system.
+
+For reference information, see [the frame system](/operate/reference/services/frame-system/).
+
+## Determine the world frame
+
+The world reference frame is the fixed coordinate system that serves as the reference point for the other frames in your robotic system.
+
+You choose the world frame, in whatever way is convenient for your application.
+It generally makes sense to define the world frame's location as a point in your space that does not move and is easy to measure from.
+
+For example, if you are using a robot arm mounted to a table, it can be convenient to decide that the world frame origin is at the base of the arm, or you can define the world frame origin as one corner of the table that the arm is mounted to.
+
+You do not explicitly configure the world frame.
+You define it implicitly by configuring the frame of a component relative to world.
+
+### Add a frame to your arm
+
+1. Mount the arm in a fixed location.
+1. On your arm's configuration card, click **+ Add frame**.
+1. The card will populate with the following default values:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "orientation": {
+       "type": "ov_degrees",
+       "value": {
+         "x": 0,
+         "y": 0,
+         "z": 1,
+         "th": 0
+       }
+     },
+     "parent": "world",
+     "translation": {
+       "x": 0,
+       "y": 0,
+       "z": 0
+     }
+   }
+   ```
+
+   This [orientation vector](/operate/mobility/orientation-vector/) of `x=0, y=0, z=1, th=0` means the frame is aligned with the world frame, and the `0, 0, 0` translation means the arm's {{< glossary_tooltip term_id="origin-frame" text="origin frame" >}} is coincident with the origin of the world frame.
+   You can think of it like this: The z-axis of the arm's origin frame is parallel to the z-axis of the world frame, and the origin of the arm's origin frame is in the same place as the origin of the world frame.
+
+   If you want to define the world frame as the corner of the table, leave the default values for now, and continue to the next step so that you know which way `x` and `y` point and can edit the configuration accordingly.
+
+1. Click **Save**.
+
+   By configuring your arm's frame like this, you have now defined the world frame as the same as the arm's base frame.
+
+### Figure out which way the arm's axes point
+
+The arm's base frame is now the world frame, but you do not know which way the x, y, and z axes of the arm frame point, unless you happen to be familiar with the arm's [kinematics file](/operate/reference/kinematic-chain-config/).
```

Comment:
This selection of text references both the arm's "origin frame" and "base frame". These are the same thing, correct? Should we stick with just one of these names or explain the different if there is one? 

It's my understanding that that origin/base frame for an arm is references the first link (usually called a base link) in the kinematics chain, which typically aligns with the world. This is opposed to the end-effector frame when configuring a component like a gripper or camera frame in reference to a parent arm. I could be incorrect in my understanding though.